master/master.cfg

```diff
@@ -602,7 +603,7 @@ def create_win_factory(os, llvm):
   else:
     # Run the tests
     env = {}
-    for hl_target in ['host', 'host-opencl', 'host-cuda']:
+    for hl_target in ['host', 'host-opencl', 'host-cuda', 'host-d3d12compute']:
```

Comment:
Not sure if it's necessary here, but should this be predicated also on it being 64-bit?

Reply:
PTAL